flexget/ui/src/plugins/series/series.module.js

```diff
@@ -7,7 +7,7 @@
     seriesModule.run(function ($state, route, sideNav, toolBar) {
         route.register('series', '/series', 'series-view');
         route.register('episodes', '/series/:id/episodes', 'series-episodes-view');
```

Comment:
I think this route can be removed, since there is no more routing to a different episodes page right?